### DISCUSSION TITLE
Error when adding a basic picker in v3 on Filament v3

### DISCUSSION THREAD

**Discussion by matt-shirey** [awcodes/filament-curator#256] [Q&A] [answered]
Hi,

I'm playing around with Filament v3 and just installed v3 of Curator but when I register the CuratorPicker on my resource and then call a basic instance of it, I get an error of "Call to undefined function ray()".  I've used this basic call before on v2 and had no issues.  Did I miss something with the v3 version?  Also, I did create a new Filament theme and added the needed css files to my admin theme css.

```
CuratorPicker::make('featured_image')
```

**Comment by awcodes** [comment, marked as answer]
Make sure your on [v3.0.0-beta.4](https://github.com/awcodes/filament-curator/releases/tag/v3.0.0-beta.4).

That was fixed and tagged this morning.

**Reply by matt-shirey** [reply]
@awcodes Thank you.  I just updated to that version and everything is good now.
